test/Kestrel.FunctionalTests/TestHelpers/TestServer.cs

```diff
@@ -79,19 +87,6 @@ public TestServer(RequestDelegate app, TestServiceContext context, ListenOptions
             _host.Start();
         }
 
-        public static void RemoveDevCert(IServiceCollection services)
-        {
-            // KestrelServerOptionsSetup would scan all system certificates on every test server creation
-            // making test runs very slow
```

Comment:
Why are you removing this? Have you measured the impact his has on test run speed on slow CI machines like Travis?

Reply:
The default cert loading was moved out of KestrelServerOptionsSetup before 2.1 RTM. I didn't remove this workaround at the time because I wasn't aware of it. I'm removing it now because it's making writing functional tests more difficult, it prevents ApplicationServices from being set so UseHttps doesn't work.

Reply:
Nice.